### PR TITLE
iOS code coming from submodule / react.force.net.js supports calls outside service/data

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: app/js/react.force.common.js [hunk 1]
```diff
@@ -1,76 +0,0 @@
-/*
```
Comment (wmathurin) [inline app/js/react.force.common.js:1]: The react.force.*.js are copied from the external/shared folder when install.sh is run, so they don't need to be checked in

File: external/android [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit 642cd44ac91b112aa6ec4f2aab08fc37cf14d450
+Subproject commit e882d0c77eef99ef90adbb77d59f00dce489fcc8
```
Comment (wmathurin) [inline external/android:1]: Latest of everything (out of unstable)

File: app/ios/PodFile [hunk 3]
```diff
@@ -21,11 +20,11 @@ pod 'React', :path => '../node_modules/react-native', :subspecs => [
     'RCTLinkingIOS'
 ]
 
-pod 'SalesforceSDKCore'
-pod 'SalesforceNetwork'
-pod 'SalesforceRestAPI'
-pod 'SmartStore'
-pod 'SmartSync'
-pod 'SalesforceReact'
+pod 'SalesforceSDKCore', :path => '../../external/ios'
```
Comment (wmathurin) [inline app/ios/PodFile:23]: That way, we can point to arbitrary versions (also it's faster)

**Comment by bhariharan** [comment]
:+1:

**Pull request merged**